src/Box.js

```diff
@@ -421,6 +421,11 @@ class Box extends Component{
 			this.props.onResizeStart && this.props.onResizeStart(e, data);
 			const startingPosition = Object.assign({}, data);
 			const movingSides = this.getMovingSides(e.target && e.target.getAttribute('id'));
+
+			let movingSidesObj = {};
+			movingSides.forEach(side => movingSidesObj[side] = true);
```

Comment:
Since there will only be 4 sides, don't time complexity for search in array compared to search in hashmap would matter much. But really pleased to see you are considering even the smallest details 👏

Reply:
Its not the 4 sides, but how many times you will be looking up a side. The movingSides array should have been an object and not an array to begin with, I think :)